recurring_contract/models/contract_group.py

```diff
@@ -437,6 +444,9 @@ def _process_invoice_generation(self, invoicer, invoicing_date, contract=None):
                     "payment_mode_id": self.payment_mode_id.id,
                 }
             )
+            open_invoice.mapped("invoice_line_ids").filtered(
+                lambda line: line.contract_id in contract_lines_to_inv.contract_id
+            ).create_analytic_lines()
```

Comment:
Why not this directly?
```suggestion
            contract_lines_to_inv.create_analytic_lines()
```